x/oracle/client/cli/query.go

```diff
@@ -419,6 +426,8 @@ func GetQueryRequestPrice() *cobra.Command {
 		},
 	}
 	flags.AddQueryFlagsToCmd(cmd)
+	cmd.Flags().Uint64(flagAskCount, 0, "The maximum number of validators that you want to respond to the request, if not provided use default value set by --with-pricer flag")
```

Comment:
The number of validators that have been chosen to respond to the request, if not provided use Band standard dataset standard.